__tests__/app.test.js

```diff
@@ -113,3 +113,59 @@ describe("GET /api/reviews", () => {
             });
     });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+    test("200: Responds with an array of comments", () => {
+        return request(app)
+            .get("/api/reviews/3/comments")
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toHaveLength(3);
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        review_id: expect.any(Number),
```

Comment:
You should definitely assert a specific review_id here as you know all comments should belong to one article.